Formula/jpeg-xl.rb

```diff
@@ -48,22 +51,19 @@ class JpegXl < Formula
         revision: "868ab558fad70fcbe8863ba4e85179eeb81cc840"
   end
 
-  resource "skcms" do
-    url "https://skia.googlesource.com/skcms.git",
-        revision: "b25b07b4b07990811de121c0356155b2ba0f4318"
-  end
-
   def install
+    ENV.append_path "XML_CATALOG_FILES", HOMEBREW_PREFIX/"etc/xml/catalog"
     resources.each { |r| r.stage buildpath/"third_party"/r.name }
-    # disable manpages due to problems with asciidoc 10
     system "cmake", "-S", ".", "-B", "build",
                     "-DJPEGXL_FORCE_SYSTEM_BROTLI=ON",
                     "-DJPEGXL_FORCE_SYSTEM_LCMS2=ON",
                     "-DJPEGXL_FORCE_SYSTEM_HWY=ON",
                     "-DJPEGXL_ENABLE_JNI=OFF",
+                    "-DJPEGXL_ENABLE_SKCMS=OFF",
                     "-DJPEGXL_VERSION=#{version}",
-                    "-DJPEGXL_ENABLE_MANPAGES=OFF",
+                    "-DJPEGXL_ENABLE_MANPAGES=ON",
                     "-DCMAKE_INSTALL_RPATH=#{rpath}",
+                    "-DPython3_EXECUTABLE=#{Formula["asciidoc"].libexec/"bin/python3"}",
```

Comment:
Do we still need this given the `docbook-xsl` dependency? What about doing something like
```ruby
ENV.prepend_path "PYTHONPATH", Formula["asciidoc"].libexec/Language::Python.site_packages("python3.11")
```
?

Reply:
This works too, I used to do that initially. Was not sure what was preferred between the two.

Also `docbook-xsl` is only used for the catalog files it installs in `HOMEBREW_PREFIX/etc/xml/catalog`.

Reply:
Hmm, I guess this is fine. Let's go with this.